### ISSUE TITLE
2C (dimensionality reduction) + 2FC network 93.6% test acc, 0.6M parameters

### ISSUE THREAD

**Issue by dtoertei** [zalandoresearch/fashion-mnist#149]
Successfully reached >93% test set accuracy with this convnet (Conv2Dsimple2) without data augmentation. 
Used stratified 3-fold train, dropout, batch normalization and max norm.

Code and description is available here: https://github.com/dtoertei/fashion-mnist
